astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,351 @@
+
+```shell
+dotnet publish -r osx-x64
+```
+
+Then start up the web application.
+
+```shell
+dotnet run
+```
+
+The output should look similar to the following.
+
+```shell
+Building...
+info: Microsoft.Hosting.Lifetime[14]
+      Now listening on: http://localhost:5146
+info: Microsoft.Hosting.Lifetime[0]
+      Application started. Press Ctrl+C to shut down.
+info: Microsoft.Hosting.Lifetime[0]
+      Hosting environment: Development
+info: Microsoft.Hosting.Lifetime[0]
+      Content root path: /Users/Shared/your-application
+```
+
+The application will be accessible in your browser at `http://localhost:{PortNumber}` where the port number will be displayed in your shell window.  I.E. `http://localhost:5146`
+
+Press Control+C in the terminal window to stop the application from running.
+
+## Authentication
+
+It's always smart to leverage existing libraries as they are likely to be more secure and better handle edge cases. You're going to add an `OpenIdConnect` library to the application. In your terminal window, make sure you're in the root directory for your application and run the command below to add the library.
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.OpenIdConnect
+```
+
+This application will contain two pages that require authorization.  The `Account` page will display the user's balance and the `MakeChange` page will make change for the user.  To protect these pages you can use the [Authorize filter attribute](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/filter?view=aspnetcore-3.1#authorize-filter-attribute) in the [page model](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-7.0&tabs=visual-studio).
+
+Create `Account.cshtml` and `Account.cshtml.cs` in the `Pages` directory. To protect the "Account" page you can use the [Authorize filter attribute](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/filter?view=aspnetcore-3.1#authorize-filter-attribute) in the [page model](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-7.0&tabs=visual-studio).
+
+Copy the following code into `Account.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Account.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `Account.cshtml`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Account.cshtml"}
+  lang="razor" />
+
+Create `MakeChange.cshtml` and `MakeChange.cshtml.cs` in the `Pages` directory. You will use the same [Authorize filter attribute](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/filter?view=aspnetcore-3.1#authorize-filter-attribute) to secure the page.
+
+Copy the following code into `MakeChange.cshtml.cs`:
```

Comment:
Why do we need the `// tag::authorize[]` comment in this file?

Reply:
We do not.  I had it there to show a subsection of code, but we decided to post whole page, so it is no longer needed.  Removed.